pkg/streamingpromql/types/data.go

```diff
@@ -98,7 +98,7 @@ type RangeVectorStepData struct {
 	StepT int64
 
 	// RangeStart is the beginning of the time range selected by this time step.
-	// RangeStart is inclusive (ie. points with timestamp >= RangeStart are included in the range).
+	// RangeStart is exclusive (ie. points with timestamp > RangeStart are included in the range).
```

Comment:
(nit) "... whilst the point with timestamp = RangeStart is excluded"

Reply:
Addressed in f3159b3.